app/src/main/res/values-es/strings.xml

```diff
@@ -86,11 +86,11 @@
     <string name="newTxFee">Impuesto:</string>
     <string name="invalidAmountAndAddress">Dirección o montos inválidos</string>
     <string name="invalidAddress">Dirección inválida</string>
-    <string name="invalidAmount">Monto inválido</string>
+    <string name="invalidAmount">Cantidad inválida</string>
     <string name="noBalanceFirstLine">Comience <a href="https://bitcoin.org/es/getting-started">a usar</a> Bitcoin</string>
     <string name="noBalanceSecondLine">Dónde <a href="http://howtobuybitcoins.info/">comprar</a> Bitcoin</string>
     <string name="noBalanceThirdLine">Lea nuestras <a href="https://greenaddress.it/es/faq">Preguntas Frecuentes</a></string>
-    <string name="trezor_headline">Por favor, siga las instrucciones en su Hardware Wallet</string>
+    <string name="trezor_headline">Por favor, siga las instrucciones en su Trezor</string>
```

Comment:
this code path supports all trezor compatible devices, so I prefer a generic word if possible like "Hardware wallet"